jest/helpers.ts

```diff
@@ -180,6 +184,13 @@ function getExecaOptions(options: SpawnOptions) {
     env.NODE_PATH = options.nodePath;
   }
 
+  if (options.env) {
+    env = {
+      ...env,
+      ...options.env,
+    };
+  }
```

Comment:
Could this be incorporated into the `Object.assign` above? (And perhaps convert that into an object spread)

Reply:
done, good catch!